src/core/room.js

```diff
@@ -131,7 +138,7 @@ export default class Room {
             });
         }
 
-        this.root.script.game.fire('join', user);
+        this.root.script[this.roomType].fire('join', user);
```

Comment:
This is not needed, game.js or any other script can subscribe on their own to `this.app.room.on...`